apps/nextjs-example/components/transactionFlow/MultiAgent.tsx

```diff
@@ -23,7 +23,7 @@ export default function MultiAgentTransaction({
     useWallet();
 
   const [secondarySignerAccount, setSecondarySignerAccount] =
-    useState<Account>();
+    useState<Ed25519Account>();
```

Comment:
this is definitely a needed change once the adapter is updated to use latest sdk version . check out https://github.com/aptos-labs/aptos-wallet-adapter/pull/296/files 😄